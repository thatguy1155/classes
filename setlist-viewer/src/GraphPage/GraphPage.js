import React, { useState, useContext, useEffect } from 'react';
import {
  Container, Row, Col,
} from 'react-bootstrap';
import { AppContext } from '../Context/Context';
import Graph from './Components/Graph';
import Loading from '../Components/Loading/Loading';
import './GraphPage.css';

export default function GraphPage(props) {
  const { artistName, isLoading } = useContext(AppContext);

  return (
    <Container fluid>
      <Row>
        <Col>
          {isLoading ? <Loading /> : <Graph />}
        </Col>
      </Row>
    </Container>
  );
}
