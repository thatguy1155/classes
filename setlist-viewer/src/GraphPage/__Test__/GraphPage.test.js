import { render, screen, fireEvent } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import renderer from 'react-test-renderer';
import GraphPage from '../GraphPage';

describe('My SearchPage', () => {
  const callback = jest.fn();
  const component = renderer.create(
    <GraphPage />,
  );

  it('should render with given state from Redux store', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });

  //   it('should show an altered name', () => {
  //     //await screen.findByText(/Signed in as/);
  //     render(<NameDisplay name={'Slim Shady'} setName={() => {}}/>);
  //     expect(screen.queryByText(/Searches for Slim Shady/)).toBeNull();

  //     userEvent.type(screen.getByRole('textbox'), 'Slim Shady');

  //     expect(
  //       screen.getByText(/my name is Slim Shady/),
  //     ).toBeInTheDocument();
  // });
});
