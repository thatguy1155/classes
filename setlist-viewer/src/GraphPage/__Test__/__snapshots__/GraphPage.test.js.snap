// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My SearchPage should render with given state from Redux store 1`] = `
<div>
  Graph Page
</div>
`;
